checking.md

```diff
@@ -3,49 +3,59 @@
-    - Credentials and other details will be provided by the organizers.
-  - Contact your DataRescue guide, if you need any assistance.
+  - Apply to become a Checker 
+    - By asking your DataRescue guide or by filling out [this form](XXX) 
+    - Skills recommended: in general, Checkers need to have an in-depth understanding of harvesting goals and potential content variations for datasets.
+    - Note that a Gmail email address is required to apply.
+    - Note also that you should be willing to have your real name be associated with the datasets, to follow archival best practices (see [Trust and archival best practices](XXX) for more information).
+  - Credentials, slack invite, Uncrawlable spreadsheet URL, and other details will be provided once your application is approved.
+  - Test the Uploader application http://drp-upload.herokuapp.com with the credentials provided
+    - Make sure to select the right event in the dropdown
+  - Verify that you have write access to the Checkers tab in the Uncrawlable spreadsheet
+  - If you need any assistance:
+      - Talk to your DataRescue Guide if you are at an in-person event
+      - Or post  questions on Slack in the Checkers channel.
 
 - Claiming a dataset for the checking step 
   - You will work on datasets that were harvested by Harvesters. 
-  - Go to the Uncrawlable spreadsheet, and look for a dataset that has the status "Harvester status indicator" = Closed
-  - Claim it by entering your slack handle with the status "Open" and today's date in the cell "Checker status indicator" in Checker section, for instance: 
+  - Go to the Uncrawlable spreadsheet, click the Checkers tab, and look for a dataset to check
+    - Available datasets are the ones whose cell "Checker Status Indicator" is empty
+    - If an item is already claimed but its "Date Opened or Closed" cell has turned red, it is also available for you to claim (for more details see the last section of this document)
+  - Claim it by entering your slack handle along with the status "Open" and today's date, for instance: 
   ```
-  @khdelphine open 1/22/2017
+  @khdelphine Open 1/22/2017
   ```
   
 - Downloading & opening the dataset
-  - Go to the URL containing the zipped dataset (in cell "URL from upload of zip", in the Harvester section) 
+  - Go to the URL containing the zipped dataset (provided in cell "URL from upload of zip") 
   - Download the zip file to your laptop, and unzip it.
 
 - Checking for completeness and meaningfulness
   - Your role is to inspect the dataset and make sure that it is complete.
   - You also need to check that the dataset is *meaningful* 
-    - This is: "will the bag make sense to a scientist"? 
-    - For instance, if a dataset is composed of a spreadsheet without any accompanying key or explanation of what the data represents, it might be completely impossible to a scientist to use it.
+    - That is: "will the bag make sense to a scientist"? 
```

Comment:
Do you mean that 'This is: "will the bag make sense to a scientist"?' should be on the same line as "... meaningful" ? Sounds good.